packages/database/src/migrations/20240807222535-AddTaskTrigger-modifies-schema.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`
+     CREATE TRIGGER task_trigger
+      AFTER INSERT OR UPDATE or DELETE
+      ON task
+      FOR EACH ROW EXECUTE PROCEDURE notification();
```

Comment:
this looks good to me. I'm not very familiar with adding hooks in the database so I'm keen to get @rohan-bes to double check that we aren't missing anything since it's our first time adding a hook to a new model.

Reply:
@alexd-bes instead of adding this trigger via a migration, it's best to add the table name to `TABLES_REQUIRING_TRIGGER_CREATION` in `runPostMigration`

That way if the trigger ever gets dropped/disabled in a migration we still re-create it automatically

Reply:
I didn't know about that one, have just updated 🙏